packages/tupaia-web/src/features/Dashboard/ExportDashboard/ExportConfig.tsx

```diff
@@ -56,12 +55,12 @@ const ExportSettingsContainer = styled.div`
   width: 100%;
   display: flex;
   flex-direction: column;
-  margin-bottom: 2rem;
+  margin-block-end: 2rem;
 
   @media screen and (min-width: ${MOBILE_BREAKPOINT}) {
     width: 62%;
-    margin-right: 2rem;
-    margin-bottom: 0;
+    margin-inline-end: 2rem;
+    margin-block-end: 0;
   }
```

Comment:
This is more of a question than anything: Do we have an established pattern for whether to:

- style for desktop, and override when on mobile; versus
- style for mobile, and override for bigger windows.

I’ve seen in a few places, like here, where desktop is the “default”; but it’s arguably more failsafe to style first for mobile, and selectively style for desktop. In theory if we’re 100% rigorous the result is identical, but this way feels a little more fail-safe